build-tools/packages/build-cli/src/test/machines/fluidReleaseMachine.test.ts

```diff
@@ -158,6 +160,7 @@ function walkExits(state: string, collector: Set<string>, step = 0): void {
 
 	const transitions = machine.list_transitions(state);
 	for (const next of transitions.exits) {
+		// eslint-disable-next-line no-param-reassign
```

Comment:
Use `step + 1` instead? Doesn't look like the function cares about having incremented it anyway other than for the recursive call.